### PR TITLE
fix device auth tests

### PR DESCRIPTION
**Pull request by jennyf19**
#2207 

TODO:
- Check which version of VS is on the windows 8 VM
- Try running the test on the VM, as it's not running correctly currently

When running locally, I now get a "this device is not compliant" error, which I think is correct, as I'm signing in with the win8 lab user on a win 10 device.

**Main fixes:**
- add `$` to file path
- Set `Microsoft.Identity.Test.Integration.Win8` to v4.6.2, as 4.7.2 is not supported on win8.
- Fix namespace (when going into properties in VS, it was showing a different default namespace)
- We need to use a specific lab user, but our lab queries are not set to filter by user. I just added the corrected user to the query, as there are no other parameters needed for this test. (before there was a mismatch in user -> password, as the default user response was returning).
- u/p does not work w/`common` so using `organizations` w/`.WithAuthority()`
- Add a reference to the MSTest.TestAdapter so the tests appear in the VS test runner
- due to newtonsoft error, the lab also needs to target net461, as 4.6.2 can only use newtonsoft v6 and lab is using v12, so there was an error popping up.

### CODE REVIEW

File: tests/Microsoft.Identity.Test.Integration.Win8/DeviceAuthenticationTests.cs [hunk 1]
```diff
@@ -21,42 +22,46 @@ public class DeviceAuthenticationTests
...
+                var (req, res) = factory.RequestsAndResponses
+                    .Where(x => x.Item1.RequestUri.AbsoluteUri == labResponse.Lab.Authority + "organizations/oauth2/v2.0/token"
+                             && x.Item2.StatusCode == HttpStatusCode.OK).ElementAt(1);
+
+                var AuthHeader = req.Headers.Single(h => h.Key == "Authorization").Value.FirstOrDefault();
 
+                Assert.IsTrue(!string.IsNullOrEmpty(AuthHeader));
+                Assert.IsTrue(AuthHeader.Contains("PKeyAuth"));
+            }
+            catch(Exception ex)
```
Comment (bgavrilMS) [inline tests/Microsoft.Identity.Test.Integration.Win8/DeviceAuthenticationTests.cs:61]: Is this needed? Exceptions fail the tests.

File: tests/Microsoft.Identity.Test.LabInfrastructure/Microsoft.Identity.Test.LabInfrastructure.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>net461; netstandard2.0</TargetFrameworks>
```
Comment (bgavrilMS) [inline tests/Microsoft.Identity.Test.LabInfrastructure/Microsoft.Identity.Test.LabInfrastructure.csproj:4]: Could we keep .net471? According to the docs, https://docs.microsoft.com/en-us/dotnet/framework/install/on-windows-8-1, this is available for Win8.1 (not sure what this means, probably Win8 + Service Pack ?) but not for Win8

**Review by bgavrilMS — CHANGES_REQUESTED** [review]
please remove try/catch assert and let's try to revert to .net471, as it is supported by win8.1 (not win8)

File: tests/Microsoft.Identity.Test.LabInfrastructure/Microsoft.Identity.Test.LabInfrastructure.csproj (hunk 2 above)
Comment (jennyf19) [reply tests/Microsoft.Identity.Test.LabInfrastructure/Microsoft.Identity.Test.LabInfrastructure.csproj:4]: I checked and the VM is 8.1, so i'll revert back on these.

File: tests/Microsoft.Identity.Test.Integration.Win8/DeviceAuthenticationTests.cs (hunk 1 above)
Comment (jennyf19) [reply tests/Microsoft.Identity.Test.Integration.Win8/DeviceAuthenticationTests.cs:61]: it was for testing. have removed.

---
In reply to: [529523144](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2235#discussion_r529523144) [](ancestors = 529523144)

**Comment by jennyf19** [comment]
done. thanks.

---
In reply to: [537466341](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2235#issuecomment-537466341) [](ancestors = 537466341)

**Review by bgavrilMS — APPROVED** [review]

**Pull request merged**